db/structure.sql

```diff
@@ -2,7 +2,7 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 13.1
+-- Dumped from database version 9.6.20
```

Comment:
I think this file was added by accident, it was generated with an older version of pg and includes some column names that are not referenced anywhere for example: `slack_name`

Reply:
I regenerated the schema file! Ill add the current version